docs/operator/getting-started.md

```diff
@@ -114,23 +94,22 @@ kubectl tree deploy nginx
 
 ```
 NAMESPACE  NAME                                                       READY  REASON  AGE
-default    Deployment/nginx                                           -              86s
-default    ├─ReplicaSet/nginx-6d4cf56db6                              -              86s
-default    │ ├─ConfigAuditReport/replicaset-nginx-6d4cf56db6          -              81s
-default    │ └─VulnerabilityReport/replicaset-nginx-6d4cf56db6-nginx  -              66s
-default    └─ReplicaSet/nginx-db749865c                               -              19s
-default      ├─ConfigAuditReport/replicaset-nginx-db749865c           -              17s
-default      ├─VulnerabilityReport/replicaset-nginx-db749865c-nginx   -              9s
-default      └─Pod/nginx-db749865c-lfcp5                              True           19s
+default    Deployment/nginx                                           -              7h5m
+default    ├─ReplicaSet/nginx-5fbc65fff                               -              2m36s
+default    │ ├─ConfigAuditReport/replicaset-nginx-5fbc65fff           -              2m36s
+default    │ ├─Pod/nginx-5fbc65fff-j7zl2                              True           2m36s
+default    │ └─VulnerabilityReport/replicaset-nginx-5fbc65fff-nginx   -              2m22s
+default    └─ReplicaSet/nginx-78449c65d4                              -              7h5m
+default      ├─ConfigAuditReport/replicaset-nginx-78449c65d4          -              5m46s
+default      └─VulnerabilityReport/replicaset-nginx-78449c65d4-nginx  -              5m22s
 ```
 </details>
 
 By following this guide you could realize that the operator knows how to attach VulnerabilityReport and
-ConfigAuditReport objects to build-in Kubernetes objects so that looking them up is easy. What's more, in this
-approach where a custom resource inherits a life cycle of the built-in resource we could leverage Kubernetes garbage
-collection. For example, when the previous ReplicaSet named `nginx-6d4cf56db6` is deleted the VulnerabilityReport named
-`replicaset-nginx-6d4cf56db6-nginx` as well as the ConfigAuditReport named `replicaset-nginx-6d4cf56db6` are
-automatically garbage collected.
+ConfigAuditReport resources to build-in Kubernetes objects. What's more, in this approach where a custom resource
+inherits a life cycle of the built-in resource we could leverage Kubernetes garbage collection. For example, when the
+previous ReplicaSet named `nginx-78449c65d4` is deleted the VulnerabilityReport named `replicaset-nginx-78449c65d4-nginx`
+as well as the ConfigAuditReport named `replicaset-nginx-78449c65d46` are automatically garbage collected.
 
 !!! tip
     If you only want the latest ReplicaSet in your Deployment to be scanned for vulnerabilities you can set the value
```

Comment:
!!! tip:
    If you only want the latest ReplicaSet in your Deployment to be scanned for vulnerabilities, you can set the value

Reply:
Why would you add the colon here? The tip keyword is recognised by mkdocs for material and will break rendering as follows:
![tipwithcolon](https://user-images.githubusercontent.com/1322923/159431387-56802396-7d66-4061-ae3e-fae756245212.png)

I can add comma in the other place, but please note that I'm not modifying this admonition in scope of this PR.